src/components/core/DpButtonIcon.vue

```diff
@@ -14,6 +14,8 @@
 </template>
 
 <script>
+import { Tooltip } from './../../directives'
```

Comment:
Here and below: import can be shortened

```suggestion
import { Tooltip } from '../../directives'
```